var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -30,13 +31,15 @@ class PyPyside2(PythonPackage):
         description="Enables the generation of html and man page documentation",
     )
 
-    depends_on("python@2.7.0:2.7,3.5.0:3.5,3.6.1:", type=("build", "run"))
-    depends_on("python@2.7.0:2.7,3.5.0:3.5,3.6.1:3.8", when="@:5.14", type=("build", "run"))
+    # dependency matrix from: https://wiki.qt.io/Qt_for_Python
+    depends_on("python@:3.10", when="@5.15.8:", type=("build", "run"))
+    depends_on("python@:3.9", when="@5.15.1:5.15.7", type=("build", "run"))
+    depends_on("python@:3.8", when="@:5.15.0", type=("build", "run"))
 
     depends_on("cmake@3.1:", type="build")
     # libclang versioning from sources/shiboken2/doc/gettingstarted.rst
-    depends_on("llvm@6", type="build", when="@5.12:5.13")
-    depends_on("llvm@10", type="build", when="@5.15")
+    depends_on("llvm@6+clang", type="build", when="@5.12:5.13")
+    depends_on("llvm@10+clang", type="build", when="@5.15")
     depends_on("py-setuptools", type="build")
     depends_on("py-packaging", type="build")
     depends_on("py-wheel", type="build")
```

Comment:
qt 5.12+ according to `sources/shiboken2/doc/gettingstarted.rst`, not sure when that changed